qtism/runtime/tests/AssessmentTestSession.php

```diff
@@ -2064,6 +2088,8 @@ public function endTestSession()
         // If there are still pending responses to be sent, apply a deffered response processing + outcomeProcessing.
         $this->defferedResponseProcessing();
 
+        $this->outcomeProcessing();
```

Comment:
If possible, could you please make sure this new step is covered by the integration/unit test scenarios?

Reply:
Right, I added a test case for it